js/jquery/fade.js

```diff
@@ -0,0 +1,123 @@
+import $ from 'jquery'
+
+const NAME = 'fade'
+const VERSION = '1.0.0-alpha.2'
+const DATA_KEY = 'axa.fade'
+const EVENT_KEY = `.${DATA_KEY}`
+const DATA_API_KEY = '.data-api'
+const JQUERY_NO_CONFLICT = $.fn[NAME]
```

Comment:
Our names don't clash with the names of the Bootstrap plugins. I'd also try to prevent having `$.modal()` and `$.axa.modal()` at any time, because it's very confusing. This is why I chose to do it like Bootstrap and replace a potentially conflicting plugin. If we ever replace a plugin, we'll be forced to completely implement the jQuery api while extending with our own features.